apps/trivium/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "tfhe-trivium"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+tfhe = { path = "../../tfhe", features = [ "boolean", "shortint", "integer" ] }
+rayon = { version = "1.7.0"}
+
```

Comment:
can you take inspiration from https://github.com/zama-ai/concrete_internal/blob/359d4c59190f0f0cb70ec6b6dba77b1c24dfea76/concrete-integer/Cargo.toml#L23 to select the hardware feature depending on platform ? :)